### PR TITLE
Feat(Stock exit) add barcode to receipts

### LINKED ISSUE #3948: Add barcodes to stock exit receipts
We should add barcodes to the stock exit documents.  See below for the current presentation:
image
_Fig 1: Stock Exit Document_

If we add barcodes, we'll be able to build a lot of functionality off of the stock modules:

1. We'll be able to add an easy look up of records in the exit registry, like https://github.com/IMA-WorldHealth/bhima/pull/3009.
2. Quickly fill out documents by scanning the stock exit (for example, to import into a depot/service).

### PR DESCRIPTION
**Pull request by jeremielodi**
closes #3948.

### CODE REVIEW

File: server/controllers/stock/reports/stock_exit_depot.receipt.handlebars [hunk 1]
```diff
@@ -20,8 +20,10 @@
             {{translate 'STOCK.RECEIPT.EXIT_DEPOT'}}
           </span> <br>
           <strong>{{exit.details.depot_name}}</strong> <br>
-          <strong>{{exit.details.document_reference}}</strong> <br>
-          <small>{{date exit.details.date}}</small> <br>
+          <div class="row">
+            <small>{{> barcode value= exit.details.barcode}}</small>
+            <br/><br/>
```
Comment (mbayopanda) [inline server/controllers/stock/reports/stock_exit_depot.receipt.handlebars]: Could you remove this `<br/><br/>`, i think it is not necessary to have these spaces

File: server/controllers/stock/reports/stock_exit_loss.receipt.handlebars [hunk 2]
```diff
@@ -19,8 +19,10 @@
           <span class="text-uppercase">
             {{translate 'STOCK.RECEIPT.EXIT_LOSS'}}
           </span> <br>
-          <span>{{details.document_reference}}</span> <br>
-          <small>{{date details.date}}</small> <br>
+          <div class="row">
+            <small>{{> barcode value=details.barcode}}</small>
+            <br/><br/>
```
Comment (mbayopanda) [inline server/controllers/stock/reports/stock_exit_loss.receipt.handlebars]: Could you remove this `<br/><br/>`, i think it is not necessary to have these spaces

File: server/controllers/stock/reports/stock_exit_patient.receipt.handlebars [hunk 3]
```diff
@@ -20,8 +20,10 @@
             {{translate 'STOCK.RECEIPT.EXIT_PATIENT'}}
           </span> <br>
           <strong>{{details.patient_reference}} - {{details.patient_display_name}}</strong> <br>
-          <span>{{details.document_reference}}</span> <br>
-          <small>{{date details.date}}</small> <br>
+          <div class="row">
+            <small>{{> barcode value=details.barcode}}</small>
+            <br/><br/>
```
Comment (mbayopanda) [inline server/controllers/stock/reports/stock_exit_patient.receipt.handlebars]: Could you remove this `<br/><br/>`, i think it is not necessary to have these spaces

File: server/controllers/stock/reports/stock_exit_service.receipt.handlebars [hunk 4]
```diff
@@ -14,23 +15,26 @@
...
           <span class="text-uppercase">
             {{translate 'STOCK.RECEIPT.EXIT_SERVICE'}}
           </span> <br>
           <strong>{{details.service_display_name}}</strong> <br>
-          <span>{{details.document_reference}}</span> <br>
-          <small>{{date details.date}}</small> <br>
-          <br>
+          <div class="row">
+            <small>{{> barcode value=details.barcode}}</small>
+            <br/><br/>
```
Comment (mbayopanda) [inline server/controllers/stock/reports/stock_exit_service.receipt.handlebars]: Could you remove this `<br/><br/>` and add one `<br>` after the `div` block before the `</h3>` tag

**Review by mbayopanda — APPROVED** [review]
LGTM 👍 

I made just some comments about `<br/><br/>`

**Review by jniles — APPROVED** [review]
LGTM.

bors r+

**Pull request merged**